.github/workflows/question-closer-debugger.yml

```diff
@@ -24,6 +24,6 @@ jobs:
           ignoreLabels: language service,internal
           involves: wardengnaw,pieandcakes,calgagi
           closeDays: 14
-          closeComment: "This issue has been closed automatically because it's labeled as a 'question' and has not had recent activity."
+          closeComment: "For housekeeping purposes, this issue has been closed because it is a question and has not had recent activity."
```

Comment:
We don't mention "housekeeping purposes" on any of the other messages, so maybe we don't need it here either.  "This issue has been closed..." is ok, I think.